.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  - push
+
+jobs:
+  all:
+    name: all
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+          registry-url: https://registry.npmjs.org
+
+      - uses: actions/cache@v2
```

Comment:
Why don't we use the built-in caching provided by `setup-node`? https://github.com/actions/setup-node#caching-packages-dependencies

Reply:
whoa cool. because I'd never seen it before.

looks like it was added 3 weeks ago:
https://github.com/actions/setup-node/releases/tag/v2.2.0

Reply:
Added to template repo https://github.com/foxglove/template-typescript/pull/32

And updated this branch too